src/devices/elko.ts

```diff
@@ -37,7 +37,7 @@ const definitions: DefinitionWithExtend[] = [
         model: '4523430',
         vendor: 'ELKO',
         description: 'ESH Plus Super TR RF PH',
-        fromZigbee: [fz.elko_thermostat, fz.thermostat],
+        fromZigbee: [fz.elko_thermostat],
```

Comment:
Why remove `fz.thermostat`? It requires a lot of code duplication in `fz.elko_thermostat`

Reply:
The reason is to have a have different handling of localTemp from that in fz.thermostat. Is there a way of having a handling of localTemp in fz.elko_thermostat taking precedence of that in fz.thermostat to avoid the code duplication?

Reply:
Yes, see https://github.com/Koenkk/zigbee-herdsman-converters/blob/347eeb3a4ce88bf7f2eb9490f8881e1d45f5e04c/src/devices/plugwise.ts#L31 as an example

Reply:
Thank you very much for the pointer! I've updated the pull request with this.